Installers/Ubuntu-Install.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+
+echo Please enter your password to continue...
+
+# Install the following dependencies from repositories without asking questions, git-core can be removed afterwords as it's only used to initialise a local git repository
+echo Installing dependencies
+sudo apt-get -y install git-core libusb-1.0.0 libudev-dev python3 cython3 python3-pip python3-pil python3-setuptools
+
+cd /tmp
+pip3 install hidapi click
+
+# Get the sudoers home directory before changing directory into it
+cd ~
+
+# If the git repository already exists then delete it
+if [ -d "python-evic" ]; then
+  rm -Rf python-evic
+fi
+
+# Clone the pythin-evic repository the change directory into it
+echo Cloning python-evic repository
+git clone git://github.com/Ban3/python-evic.git && cd ./python-evic
+
+# Install python-evic
+echo Installing python-evic
+sudo python3 setup.py install
+
+# Cleanup
+cd ../
+rm -Rf python-evic
+
+echo Installation complete. Run \"sudo evic-usb upload /path/to/firmware.bin\" to install a new firmware.
```

Comment:
Could the capitalization also be removed? using `installers` instead of `Installers`